config/environments/production.rb

```diff
@@ -45,21 +44,31 @@
   # config.action_cable.url = "wss://example.com/cable"
   # config.action_cable.allowed_request_origins = [ "http://example.com", /http:\/\/example.*/ ]
 
+  # Assume all access to the app is happening through a SSL-terminating reverse proxy.
+  # Can be used together with config.force_ssl for Strict-Transport-Security and secure cookies.
+  # config.assume_ssl = true
+
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  # config.force_ssl = true
+  config.force_ssl = true
 
-  # Include generic and useful information about system operation, but avoid logging too much
-  # information to avoid inadvertent exposure of personally identifiable information (PII).
-  config.log_level = :info
+  # Log to STDOUT by default
+  config.logger = ActiveSupport::Logger.new(STDOUT)
```

Comment:
do the logging changes in Rails 7.1 mean we no longer get log/production.log? I see mention of a broadcast logger.

Reply:
looks like it
https://github.com/rails/rails/commit/2b1fa89e442ecb99e262a34eb11bc1110912cd49
https://github.com/rails/rails/issues/50369

Reply:
ElasticBeanstalk has an option to request logs from web console either last 100 lines or full logs. Its expecting there to be a production.log so now fails:

```
2024/06/28 08:21:39.447424 [ERROR] An error occurred during execution of command [tail-log] - [GetTailLogs]. Stop running the command. Error: fail to check stat of file /var/app/containerfiles/logs/production.log: stat /var/app/containerfiles/logs/production.log: no such file or directory 
```

So to preserve this functionality we either need to find a workaround for EB or add the production.log back in